hw/top_earlgrey/dv/chip_rom_tests.hjson

```diff
@@ -11,7 +11,7 @@
     {
       name: rom_e2e_smoke
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["//sw/device/silicon_creator/rom/e2e:rom_e2e_smoke:1:signed"]
+      sw_images: ["//sw/device/silicon_creator/rom/e2e:empty_test_slot_a:1:signed:ot_flash_binary"]
```

Comment:
What is `empty_test_slot_a` flag supposed to do?

Reply:
that's not a flag, thats part of the bazel target label. Ever since we moved to Bazel, everything after the second colon are indicies / tags.